test/e2e/sync/sync.go

```diff
@@ -441,35 +418,39 @@ var _ = CMDDescribe("pvsadm image sync tests", func() {
 		status, stdout, stderr := runSyncCMD(
 			"--help",
 		)
-		Expect(status).To(Equal(0))
-		Expect(stderr).To(Equal(""))
+		Expect(stderr).To(BeEmpty())
+		Expect(status).To(BeZero())
 		Expect(stdout).To(ContainSubstring("Examples:"))
 	})
 
 	framework.NegativeIt("run without spec-file flag", func() {
 		status, _, stderr := runSyncCMD()
-		Expect(status).NotTo(Equal(0))
+		Expect(status).NotTo(BeZero())
 		Expect(stderr).To(ContainSubstring(`"spec-file" not set`))
 	})
 
 	framework.NegativeIt("run with yaml file that doesn't exist", func() {
 		status, _, stderr := runSyncCMD("--spec-file", "fakefile.yaml")
-		Expect(status).NotTo(Equal(0))
+		Expect(status).NotTo(BeZero())
 		Expect(stderr).To(ContainSubstring(`no such file or directory`))
 	})
 
 	It("Copy Object Between Buckets", func() {
+		var err error
+		bxCli, err = client.NewClientWithEnv(APIKey, client.DefaultEnv, debug)
```

Comment:
wondering this can be moved to before block so that in future we can use this bxCli in multiple `It` blocks